network/wsNetwork.go

```diff
@@ -1956,6 +1978,57 @@ func (wn *WebsocketNetwork) prioWeightRefresh() {
 	}
 }
 
+func primaryNetwork(network protocol.NetworkID) string {
+	return fmt.Sprintf("algorand-%s.network", network)
+}
+
+func secondaryNetwork(network protocol.NetworkID) string {
+	return fmt.Sprintf("algorand-%s.net", network)
+}
```

Comment:
This looks similar to the host dedupe mask we were talking about in Slack. These patterns could be a configuration alongside the bootstrapID. This could also let you avoid the primary/secondary special case and treat all SRV handling uniformly as bootstrap/mask. If the bootstrap provider doesn't provide a mask, then de-duplication would be implicitly disabled (or treated as an error).

The approach here also breaks down if the dns bootstrap URLs are provided in the wrong order, because the network/net masks would be incorrect.